svir/dialogs/viewer_dock.py

```diff
@@ -434,15 +561,28 @@ def draw_dmg_total(self):
         '''
         Plots the total damage distribution
         '''
-        rlz = int(self.rlz_cbx.currentText().split('-')[1])  # from rlz-xxx
-        loss_type = self.loss_type_cbx.currentText()
-        means = self.dmg_total['array'][rlz][loss_type]['mean']
-        stddevs = self.dmg_total['array'][rlz][loss_type]['stddev']
+        if len(self.dmg_total['array']) == 0:
+            msg = 'No assets satisfy the selected criteria'
+            log_msg(msg, level='W', message_bar=self.iface.messageBar())
+            self.plot.clear()
+            self.plot_canvas.draw()
+            return
+        rlz = self.rlz_cbx.currentIndex()
+        # TODO: re-add error bars when stddev will become available again
+        # loss_type = self.loss_type_cbx.currentText()
+        # means = self.dmg_total['array'][rlz][loss_type]['mean']
+        # stddevs = self.dmg_total['array'][rlz][loss_type]['stddev']
+        means = self.dmg_total['array'][rlz]
+        if any(means < 0):
```

Comment:
Just for your information: since `means` is a numpy array, it is more efficient to use `(means < 0).any()` (not that it will make any difference in this case).

Reply:
Done